bin/run_cypress.py

```diff
@@ -0,0 +1,65 @@
+"""
+Run the Cypress tests
+"""
+import logging
+import os
+from pathlib import Path
+import shutil
+from subprocess import run, Popen, CalledProcessError, PIPE
+import sys
+
+logging.basicConfig()
+_log = logging.getLogger("run_cypress")
+_log.setLevel(logging.INFO)
+
+
+# noinspection SpellCheckingInspection
+def main():
+    cwd = Path(os.path.curdir).absolute()
```

Comment:
```suggestion
    cwd = Path.cwd()
```

https://docs.python.org/3/library/pathlib.html#pathlib.Path.cwd